### PR TITLE
Update french.yml

### PR DESCRIPTION
**Pull request by Mitsu13Ion**
#### Description: 
It's a full french translation for towny
____
- [ ] I have tested this pull request for defects on a server. 

By making this pull request, I represent that I have the right to waive copyright and related rights to my contribution, and agree that all copyright and related rights in my contributions are waived, and I acknowledge that the TownyAdvanced organization has the copyright to use and modify my contribution under the Towny [License](https://github.com/LlmDl/Towny/blob/master/LICENSE.md) for perpetuity.

### CODE REVIEW

File: resources/french.yml [hunk 1]
```diff
@@ -866,45 +866,45 @@ townboard_message_colour_2: '&e'
...
-msg_err_king_of_that_nation_is_not_online: '&bThe King of %s, %s, is not online. Merge failed.'
-nation1_has_merged_with_nation2: '&6The nation %s has agreed to merge together with the prevailing nation of %s.'
-msg_would_you_merge_your_nation_into_other_nation: '&6Would you agree to dissolve your nation %s into the nation of %s? This will transfer all towns and nation bank money over to %s.'
-no_open_towns: 'No open towns to list.'
+msg_error_invalid_comparator_nation: '&bComparateur invalide. Essayez resident, balance, towns, name, townblocks, online.'
+msg_error_invalid_comparator_town: '&bComparateur invalide. Essayez resident, balance, name, townblocks, online, open.'
+msg_err_merging_for_kings_only: '&bSeul un roi peut fusionner.'
+msg_err_king_of_that_nation_is_not_online: '&bLe Roi de %s, %s, n''est pas en ligne. fusionnement échoué.'
+nation1_has_merged_with_nation2: '&6La nation %s a accepté de fusionner avec la nation de %s.'
+msg_would_you_merge_your_nation_into_other_nation: '&6Voulez vous accepter de fusionner votre nation %s avec la nation de %s? Cela transférera tout l'argent des villes et de la nation à la nation de %s.'
```
Comment (LlmDl) [inline resources/french.yml]: you must add a second ' to l'argent.

File: resources/french.yml [hunk 2]
```diff
@@ -866,45 +866,45 @@ townboard_message_colour_2: '&e'
...
 # Added in 0.52:
 msg_error_nation_has_no_allies: 'La Nation n''a pas d''alliés.'
 msg_error_nation_has_no_enemies: 'La Nation n''a pas d''ennemis.'
-msg_error_ta_plot_claim: 'Missing playername, ex: /ta plot claim PlayerNameHere'
+msg_error_ta_plot_claim: 'nom de joueur manquant, ex: /ta plot claim NomDuJoueur'
 msg_error_no_player_with_that_name: 'Aucun joueur trouvé avec le nom %s'
 msg_admin_has_given_you_a_plot: 'Un admin a revendiqué une parcelle pour vous en %s'
-msg_town_forcepvp_setting_set_to: 'The town %s has had their pvp setting forced to %s'
-msg_nation_changed_open: '&cThe Nation''s open status is now %s.'
+msg_town_forcepvp_setting_set_to: 'La ville %s a l'option de pvp forcé en mode %s'
```
Comment (LlmDl) [inline resources/french.yml]: you must add a second ' to l'option.

File: resources/french.yml [hunk 3]
```diff
@@ -866,45 +866,45 @@ townboard_message_colour_2: '&e'
...
+msg_nation_changed_open: '&cL'accès à la nation est maintenant %s.'
```
Comment (LlmDl) [inline resources/french.yml]: A second ' to L'acces

File: resources/french.yml [hunk 4]
```diff
@@ -925,92 +925,92 @@ war_hud_nation: 'Nation: '
...
-outpost: 'outpost'
-msg_err_not_same_town: '&b%s doesn''t belong to your town.'
+msg_err_days_must_be_greater_than_zero: '&cLe nombre de jour doit être plus grand que 0.'
+msg_you've_been_jailed_for_x_days: 'Vous avez été emprisoné pour %s jours.'
+msg_jailed_for_x_days: ' &2pour %s jours.'
+msg_mysql_dump_success: 'Dump de base de donnée MySQL complété.'
+msg_err_mysql_not_being_used: 'Votre configuration n''est pas faite pour sauvegarder et charger du mysql.'
+msg_save_success: 'Base de donnée sauvegardée.'
+msg_load_success: 'Base de donnée chargée.'
+msg_err_cannot_afford_to_set_outpost: 'Votre ville n''a pas assez d'argent pour définir ce bloc de ville en tant qu''avant-poste.'
```
Comment (LlmDl) [inline resources/french.yml]: second ' to d'argent

File: resources/french.yml [hunk 5]
```diff
@@ -925,92 +925,92 @@ war_hud_nation: 'Nation: '
...
-msg_err_location_is_not_within_a_town: 'Location is not within a town.'
-msg_err_location_is_not_within_a_jail_plot: 'Location is not within a Jail plot.'
+msg_err_this_plot_doesnt_have_any_associated_metadata: 'Cette parcelle n''a aucune metadonnées associées.'
+msg_err_the_metadata_for_key_is_not_registered: 'Les métadonnées pour ''%s'' ne sont pas enregistrées!'
+msg_key_x_was_successfully_updated_to_x: 'La clef %s a été changée pour %s avec succès.'
+msg_err_key_x_is_not_part_of_this_plot: 'La clef %s ne fait pas parti de cette parcelle, si possible veuillez l''ajouter.'
+msg_err_key_x_already_exists: 'La clef %s existe déjà!'
+msg_custom_data_was_successfully_added_to_townblock: 'Les données custom on bien été ajoutées au bloc de ville!'
+msg_data_successfully_deleted: 'Donnée effacées avec succès!'
+msg_err_key_cannot_be_deleted: 'La clef ne peut pas être supprimée car elle n'existe pas!'
```
Comment (LlmDl) [inline resources/french.yml]: second ' to n'existe

File: resources/french.yml [hunk 6]
```diff
@@ -925,92 +925,92 @@ war_hud_nation: 'Nation: '
...
 msg_war_prefix: '&6[Guerre]&f '
-msg_war_participants_header: '&6[Guerre] &eNation Name &f(&bTowns&f)'
+msg_war_participants_header: '&6[Guerre] &eNom de la nation &f(&bTowns&f)'
 msg_war_participants: '&6[War] &e%s &f(&b%s&f)'
 
 # Added in 0.61
-msg_war_flag_deny_town_under_attack: 'You cannot do that while under attack!'
-msg_war_flag_deny_nation_under_attack: 'You cannot do this while a town in your nation is under attack!'
+msg_war_flag_deny_town_under_attack: 'Vous ne pouvez pas faire ça tout en étant attaqué!'
+msg_war_flag_deny_nation_under_attack: 'Vous ne pouvez pas faire ça pendant qu'une ville ou une nation est sous attaque!'
```
Comment (LlmDl) [inline resources/french.yml]: second ' to qu'une

File: resources/french.yml [hunk 7]
```diff
@@ -1032,48 +1032,48 @@ msg_err_cannot_unclaim_homeblock: 'Tu ne peux pas désannexer ton centre-ville,
...
+msg_plot_group_claim_confirmation: 'Cette parcelle fait parti d''un groupe de %s parcelle(s) En confirmant votre annexion, vous achèterez toutes les parcelles.'
+msg_plot_group_unclaim_confirmation: 'Cette parcelle fait parti d''un groupe de %s parcelle(s) En confirmant vous les désannexerez toutes.'
+msg_plot_group_toggle_confirmation: 'Cette parcelle fait parti d''un groupe de %s parcelle(s) En confirmant vous activerez une propriétée sur toutes.'
+msg_plot_group_set_perm_confirmation: 'Cette parcelle fait parti d''un groupe de %s parcelle(s) En confirmant vous définirez la ligne de permission sur toutes.'
+msg_plot_evict_group: 'Vous avez expulsé ce joueur de ce groupe de parcelles, %s.'
+msg_err_plot_belongs_to_group_plot_fs: 'Parcelle: %s appartien à un groupe, utilisez /plot group [nom du groupe] fs [nombre]'
+msg_err_plot_belongs_to_group_set: 'La parcelle appartient à un groupe, Utilisez /plot group set'
+msg_plot_group_set_for_sale: 'Le groupe %s a été mis en vente!'
+msg_err_plot_already_belongs_to_a_group: 'Ce plot appartient déjà à un groupe: %s, veuillez retirer ce bloc de ville de ce groupe avant d'en ajouter un autre.'
```
Comment (LlmDl) [inline resources/french.yml]: second ' to d'en

**Comment by Mitsu13Ion** [comment]
Sorry and it's good, normally

**Pull request merged**